<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProgressDialog</class>
 <widget class="QDialog" name="AddProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="disciplineBox"/>
       </item>
       <item>
        <widget class="QComboBox" name="lecturerBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="scoreLine">
         <property name="placeholderText">
          <string>Оценка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dateLine">
         <property name="placeholderText">
          <string>Дата оценки</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>68</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addProgressButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
